src/components/ColorInputFlyout/ColorPickerFlyout.tsx

```diff
@@ -27,14 +27,26 @@ export const ColorPickerFlyout: FC<ColorPickerFlyoutProps> = ({
     const [open, setOpen] = useState(false);
     const [currentFormat, setCurrentFormat] = useState(ColorFormat.Hex);
 
+    const handleOpenChange = (isOpen: boolean) => {
+        if (!isOpen) {
+            onClose?.();
+        }
+        setOpen(isOpen);
+    };
+
+    const handleClick = () => {
+        setOpen(false);
+        onClick && onClick();
```

Comment:
```suggestion
        onClick?.();
```
according to `onClose`-call

Reply:
Fixed